manifests/piped/values.yaml

```diff
@@ -65,6 +65,9 @@ secret:
     decryptServiceAccount:
       fileName: sealed-secret-gcpkms-decrypt-service-account
       data: ""
+  awsCredentials:
```

Comment:
Should we put this inside an `imageProvider` block?

Reply:
You intend to clearly distinguish from other use cases (e.g. Lambda Deployment), right? If so, agree with you!

Reply:
Yes. That is the point.

Reply:
Okay 👍

Reply:
Then we will have credentials for each use-case for specific requirements? 🤔

Reply:
Hmm... on second thought I'm begging to feel we don't have to distinguish them as @khanhtc1202  may consider. In the general case, a piped operator prepares an AWS IAM user for piped and attach needed roles. I'd say it's an edge case to separate users by features.

Reply:
(FYI: the Go SDK for AWS is load credentials from the environment variables. It must be shared in a single process.)

Reply:
Got it. 👍